src/ReservationItem.php

```diff
@@ -554,6 +554,21 @@ public static function showListSimple()
             'display_emptychoice' => true,
         ]);
 
+        echo "</td></tr>";
+
+        // Location dropdown
+        $user = new User();
+        $user->getFromDB(Session::getLoginUserID());
+        $entities = Profile_User::getUserEntities($user->fields['id'], true);
+        $locrand = mt_rand();
+        echo "<tr class='tab_bg_1'><td><label for='dropdown_locations_id$locrand'>" . __('Item location') . "</label></td><td>";
+        Location::dropdown([
+            // Fill with submitted data if any, otherwise use user's location
+            'value'  => $_POST['locations_id'] ?? $user->fields['locations_id'],
+            'rand'   => $locrand,
+            'entity' => count($entities) <= 0 ? -1 : $entities,
```

Comment:
Maybe filtering could be directly done on active entities.

```suggestion
            'entity' => $_SESSION['glpiactiveentities'],
```

Reply:
`$_SESSION['glpiactiveentities']` reflects the entities of the "desired entity" selector? If so I agree it would be better to use it.

Reply:
Indeed, it reflects the selected entities in the entities selector.